src/constants/GradeConstants.tsx

```
@@ -0,0 +1,237 @@
+import type { TotalGradeItem } from '@/types/grade';
+import TermGrade from '@/components/grade/TermGrade';
+import TotalGrade from '@/components/grade/TotalGrade';
+
+// Grade 탭 관련 상수
+export const GRADE_TABS = [
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)